src/content/docs/en/reference/api-reference.mdx

```diff
@@ -206,28 +214,30 @@ Astro.response.headers.set('Set-Cookie', 'a=b; Path=/;');
 
 <p><Since v="1.4.0" /></p>
 
+**Type:** `AstroCookies`
+
 `Astro.cookies` contains utilities for reading and manipulating cookies in [server-side rendering](/en/guides/server-side-rendering/) mode.
 
 ##### `get`
 
 <p>
-**Type:** `(key: string, options?: CookieGetOptions) => AstroCookie`
+**Type:** `(key: string, options?: CookieGetOptions) => AstroCookie | undefined`
```

Comment:
The real type is `AstroCookieGetOptions`. Was the type renamed at some point (I couldn't find a reference) or was it to shorten the type declared in the documentation? (I found other examples where the `Astro` prefix is ​​missing)

Reply:
Again, I would say let’s use the full name. Very probable something was renamed and these docs were never updated.